src/www/js/form.js

```diff
@@ -1,14 +1,15 @@
+/* eslint-disable no-empty-label */
 /* global $ */
-"use strict";
+'use strict';
 
-
-var generateProgressBar, generateProgressVal, uploadProgressBar, uploadProgressVal, statusText;
-var uploadFinished = false;
-
-var isCancelling = false;
+let generateProgressBar, generateProgressVal, uploadProgressBar, uploadProgressVal, statusText, date, percent, customMenuButton, menuContent, fontAwesomeIcom, msg;
+let uploadFinished = false;
+let isCancelling = false;
+let generationStarted = false;
+let initialValue = 0;
 
 function updateGenerateProgress(perc) {
-  generateProgressBar.animate({'width': perc + '%'}, 200, 'linear', function () {
+  generateProgressBar.animate({'width': perc + '%'}, 200, 'linear', function() {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Block must be padded by blank lines. (padded-blocks)](https://app.codacy.com/manual/triplez/open-event-webapp/pullRequest?prid=3226522)